### PR TITLE
W-10863931: HTTP_Request_configuration not considering the proxy settings

### PR DESCRIPTION
**Pull request by alopezbizon**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -17,7 +17,7 @@
     <version>1.7.0-SNAPSHOT</version>
 
     <properties>
-        <asyncHttpClientVersion>1.14-MULE-021</asyncHttpClientVersion>
+        <asyncHttpClientVersion>1.14-MULE-022-SNAPSHOT</asyncHttpClientVersion>
```
Comment (fsgonz) [inline pom.xml:20]: Leave this to the latest release which is 1.14-MULE-022. This probably only works locally.

File: src/test/java/org/mule/service/http/impl/service/client/GrizzlyHttpsClientProxyTestCase.java [hunk 2]
```diff
@@ -0,0 +1,127 @@
+package org.mule.service.http.impl.service.client;
+
+import static org.mule.runtime.http.api.HttpConstants.HttpStatus.INTERNAL_SERVER_ERROR;
+import static org.mule.runtime.http.api.HttpConstants.HttpStatus.OK;
+import static org.mule.runtime.http.api.HttpConstants.Method.POST;
+import static org.mule.runtime.http.api.HttpConstants.Protocol.HTTP;
```
Comment (fsgonz) [inline src/test/java/org/mule/service/http/impl/service/client/GrizzlyHttpsClientProxyTestCase.java:6]: import order.

File: src/test/java/org/mule/service/http/impl/service/client/GrizzlyHttpsClientProxyTestCase.java [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+      proxyServer.stop();
+    }
+    if (client != null) {
+      client.stop();
+    }
+  }
+
+  @Override
+  protected HttpResponse setUpHttpResponse(HttpRequest request) {
+    HttpResponseBuilder response = HttpResponse.builder();
```
Comment (fsgonz) [inline src/test/java/org/mule/service/http/impl/service/client/GrizzlyHttpsClientProxyTestCase.java:91]: verify here that no incorrect headers are passed.

**Review by fsgonz — CHANGES_REQUESTED** [review]

File: src/test/java/org/mule/service/http/impl/service/client/GrizzlyHttpsClientProxyTestCase.java (hunk 2 above)
Comment (alopezbizon) [reply src/test/java/org/mule/service/http/impl/service/client/GrizzlyHttpsClientProxyTestCase.java:6]: I couldn't figure out what order is wrong. I put (1) mule static imports, (2) java static imports, (3) other static imports, (4) mule imports and (5) other imports as there aren't java imports. Please, let me know where is my mistake. Thanks!

File: src/test/java/org/mule/service/http/impl/service/client/GrizzlyHttpsClientProxyTestCase.java (hunk 3 above)
Comment (alopezbizon) [reply src/test/java/org/mule/service/http/impl/service/client/GrizzlyHttpsClientProxyTestCase.java:94]: Assertion added.

File: src/test/java/org/mule/service/http/impl/service/client/GrizzlyHttpsClientProxyTestCase.java (hunk 2 above)
Comment (fsgonz) [reply src/test/java/org/mule/service/http/impl/service/client/GrizzlyHttpsClientProxyTestCase.java:6]: My mistake! I was thinking of another PR when you send me this. Sorry. 😅

**Review by fsgonz — DISMISSED** [review]

**Review by fsgonz — APPROVED** [review]

**Pull request merged**